tests/ballerina-spec-conformance-tests/build.gradle

```diff
@@ -50,5 +50,4 @@ test {
 configurations {
     testImplementation.exclude group: 'org.slf4j', module: 'slf4j-log4j12'
     testImplementation.exclude group: 'org.codehaus.woodstox', module: 'woodstox-core-asl'
-    testImplementation.exclude group: 'org.codehaus.woodstox', module: 'stax2-api'
```

Comment:
Do we know why this is an issue only with conformance tests now? For example, this change wasn't required for unit tests?

Reply:
Getting the exception from here https://github.com/ballerina-platform/ballerina-lang/blob/2d88caf6160a38acc1e86d129dbb476be5b9da0a/bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/XmlNonElementItem.java#L192 as `io.ballerina.runtime.internal.util.exceptions.BallerinaException {"message":"failed to get xml as string: org.codehaus.stax2.XMLOutputFactory2"}`

Reply:
Yeah, but my question was why and why now?

Reply:
Merging this PR for now since the conformance tests are failing without this. Please address once the tuple changes are done. Created https://github.com/ballerina-platform/ballerina-lang/issues/39201 also.